vendor/zkllvm-metacraft-circuits/src/circuit_utils/circuit_byte_utils.h

```diff
@@ -69,7 +61,9 @@ namespace circuit_byte_utils {
         static_assert(N <= InputSize);
         assert_true(N + offset <= InputSize);
         static_vector<Byte, N> ret;
-        copy(val.begin() + offset, val.begin() + offset + N, ret.begin());
+        for (size_t i = 0; i < N; i++) {
+            *(ret.begin() + i) = *(val.begin() + offset + i);
```

Comment:
Check if this workaround is fixed in the most up to date version of zkllvm

Reply:
We need this, because static_vector is packed and we don't have operator[] implemented for it. This is because we don't want to return a reference to unaligned memory.